docs/faq.md

```diff
@@ -77,4 +77,13 @@ server {
     rewrite ^/oidc/callback /index.php?module=LoginOIDC&action=callback&provider=oidc redirect;
     # ...
 }
+
+- [Unikname Connect](https://unikname.com)
+
+  - Name: `Connect with your private @unikname`
+  - Authorize URL: `https://connect.unikname.com/oidc/authorize`
+  - Token URL: `https://connect.unikname.com/oidc/accessToken`
+  - Userinfo URL: `https://connect.unikname.com/oidc/profile`
+  - Userinfo ID: `sub`
+  - OAuth Scopes: `openid email`
```

Comment:
This part is written in the nginx code block. Can you move it between Gitlab and Microsoft Azure AD?

Reply:
Moved between Gitlab and Azure